### PR TITLE
live.nicovideo.jp

### LINKED ISSUE #107456: live.nicovideo.jp
### Issue URL (Social Widget)
 [https://live.nicovideo.jp/watch/lv335022643](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Flive.nicovideo.jp%2Fwatch%2Flv335022643)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.9 nightly 4
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode: | disabled
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Russian, <br/>AdGuard Chinese
Userscripts: | https://userscripts.adtidy.org/beta/assistant/4.3/assistant.user.js

### PR DESCRIPTION
**Pull request by SKEIDs**
Fix #107456 

<details><summary>screenshot</summary>

</details><br/>

### CODE REVIEW

**Comment by gork7777** [comment]
your `##ul[class^="___menu-dock___"]` rule is blocking other useful elements.
Let's use these and these are better I guess.

For social: `live.nicovideo.jp##a[class*="local-anchor"]`
For banner: `live.nicovideo.jp##a[class*="app-download-banner"]`

**Comment by SKEIDs** [comment]
> 

OK. Thank you for tell me the rule.

**Comment by SKEIDs** [comment]
@gork7777 
 OK. Thank you for tell me the rule.

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 1]
```diff
@@ -15,6 +15,7 @@
 ##head[prefix~="medium-com:"] ~ body .js-openInAppButton
 ##head[prefix~="medium-com:"] ~ body .js-postActionsBarContent > .js-readNextInteractions
 !
+live.nicovideo.jp##a[class*="app-download-banner"]
```
Comment (Yuki2718) [inline AnnoyancesFilter/sections/mobile-app.txt:18]: Better `live.nicovideo.jp##a[class^="___app-download-banner"]`

File: JapaneseFilter/sections/specific.txt [hunk 2]
```diff
@@ -2475,6 +2475,7 @@ sp.nicovideo.jp##.siteWide-BannerAd
 dic.nicovideo.jp###right-column > div[style*="height:250px; width:300px;"]
 dic.nicovideo.jp##.ad-bannar-maincolumn-top
 dic.nicovideo.jp##.adsense-728
+live.nicovideo.jp##aside[class*="banner-ad"]
```
Comment (Yuki2718) [inline JapaneseFilter/sections/specific.txt:2478]: `live.nicovideo.jp##aside[class^="___banner-ad"]`

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

**Comment by SKEIDs** [comment]
sorry, Why is "___" necessary?

**Comment by Yuki2718** [comment]
> sorry, Why is "___" necessary?

The point is rather `^=` is prefered to `*=` as is more accurate and more efficient.

**Comment by SKEIDs** [comment]
> The point is rather ^= is prefered to *= as is more accurate and more efficient.

I see . thank you.

**Comment by SKEIDs** [comment]
> If they changed the class name so that `^=` doesn't apply any more, it's good to switch to `*=`.

I see. that is good.

**Comment by Yuki2718** [comment]
But for now use `^=`

**Review by Yuki2718 — APPROVED** [review]

**Comment by Yuki2718** [comment]
Whoops, this one `live.nicovideo.jp##a[class^="local-anchor"]` doesn't work. Revert to the original your filter.

**Comment by SKEIDs** [comment]
> Whoops, this one `live.nicovideo.jp##a[class^="local-anchor"]` doesn't work. Revert to the original your filter.

OK. Is it `*=` except `live.nicovideo.jp##a[class^="local-anchor"]`?

**Comment by Yuki2718** [comment]
Sorry I was not clear but you figured it out correctly.

**Comment by SKEIDs** [comment]
> Sorry I was not clear but you figured it out correctly.

thank you so much.

**Review by zloyden — APPROVED** [review]

**Pull request merged**